packages/addons/service/snapclient/source/resources/language/English/strings.po

```diff
@@ -0,0 +1,68 @@
+# Kodi Media Center language file
```

Comment:
```
# Kodi Media Center language file
# Addon Name: snapclient
# Addon id: service.snapclient
# Addon Provider: Team LibreELEC
msgid ""
msgstr ""
```
to follow the standard (what Kodi is doing)

Reply:
done